SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialOutputDevice.cpp

```diff
@@ -14,6 +14,9 @@ FSpatialOutputDevice::FSpatialOutputDevice(USpatialWorkerConnection* InConnectio
 	const TCHAR* CommandLine = FCommandLine::Get();
 	bLogToSpatial = !FParse::Param(CommandLine, TEXT("NoLogToSpatial"));
 
+	// Check if logging to spatial is disabled in the GUI
+	bLogToSpatialBox = !GetDefault<USpatialGDKSettings>()->SpatialLogging;
```

Comment:
You could encapsulate this all in bLogToSpatial and not need to introduce a new variable